lib/rubocop/ast/node/mixin/method_identifier_predicates.rb

```diff
@@ -12,6 +12,8 @@ module MethodIdentifierPredicates
                               map reduce reject reject! reverse_each select
                               select! times upto].freeze
 
+      ENUMERABLE_METHODS = (Enumerable.instance_methods + [:each]).freeze
```

Comment:
There's a small chance this might cause an issue due to a mismatch between Ruby version RuboCop is run with versus the version of the code inspected, but I think it's a small enough concern that this is still okay.

Reply:
That's a good point. Might be best to list them explicitly.